ddtrace/contrib/grpc/aio_client_interceptor.py

```diff
@@ -52,11 +53,15 @@ def _handle_add_callback(call, callback):
         callback(call)
 
 
-def _done_callback(span, code, details):
-    # type: (Span, grpc.StatusCode, str) -> Callable[[aio.Call], None]
+def _done_callback(span, code=None, details=None):
     def func(call):
-        # type: (aio.Call) -> None
+        nonlocal code, details
         try:
+            # type: (aio.Call) -> None
+            if not code and not details:
+                # we need to call __repr__ since we cannot call code() or details() since they are both async
+                code, details = parse_rpc_repr_string(call.__repr__())
```

Comment:
keep changes internal. I would also catch an exception from this and log the error.

```suggestion
                code, details = _parse_rpc_repr_string(call.__repr__())
```